From 335749a33c02ec0dc7ebadb9eda101d174ad5faf Mon Sep 17 00:00:00 2001
From: shezm <505306252@qq.com>
Date: Thu, 14 Apr 2022 23:20:28 +0800
Subject: [PATCH 4663/5664] HIVE-25416 : Hive metastore memory leak because
 datanucleus-api-jdo bug (#2555)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* HIVE-25416 : Hive metastore memory leak cause datanucleus-api-jdo bug
Co-authored-by: 佘志铭 <shezhiming@GIH-L-4622.local>
---
 pom.xml                      | 6 +++---
 standalone-metastore/pom.xml | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/pom.xml b/pom.xml
index 0539469..e4000c4 100644
--- a/pom.xml
+++ b/pom.xml
@@ -108,10 +108,10 @@
     <avro.version>1.8.2</avro.version>
     <bcprov-jdk15on.version>1.64</bcprov-jdk15on.version>
     <calcite.version>1.25.0</calcite.version>
-    <datanucleus-api-jdo.version>5.2.4</datanucleus-api-jdo.version>
-    <datanucleus-core.version>5.2.4</datanucleus-core.version>
+    <datanucleus-api-jdo.version>5.2.8</datanucleus-api-jdo.version>
+    <datanucleus-core.version>5.2.10</datanucleus-core.version>
     <datanucleus-jdo.version>3.2.0-release</datanucleus-jdo.version>
-    <datanucleus-rdbms.version>5.2.4</datanucleus-rdbms.version>
+    <datanucleus-rdbms.version>5.2.10</datanucleus-rdbms.version>
     <commons-cli.version>1.4</commons-cli.version>
     <commons-codec.version>1.15</commons-codec.version>
     <commons-collections.version>3.2.2</commons-collections.version>
diff --git a/standalone-metastore/pom.xml b/standalone-metastore/pom.xml
index b5ee81f..8b24226 100644
--- a/standalone-metastore/pom.xml
+++ b/standalone-metastore/pom.xml
@@ -61,10 +61,10 @@
     <commons-lang3.version>3.9</commons-lang3.version>
     <commons-logging.version>1.1.3</commons-logging.version>
     <commons-dbcp2.version>2.7.0</commons-dbcp2.version>
-    <datanucleus-api-jdo.version>5.2.4</datanucleus-api-jdo.version>
-    <datanucleus-core.version>5.2.4</datanucleus-core.version>
+    <datanucleus-api-jdo.version>5.2.8</datanucleus-api-jdo.version>
+    <datanucleus-core.version>5.2.10</datanucleus-core.version>
     <datanucleus-jdo.version>3.2.0-release</datanucleus-jdo.version>
-    <datanucleus-rdbms.version>5.2.4</datanucleus-rdbms.version>
+    <datanucleus-rdbms.version>5.2.10</datanucleus-rdbms.version>
     <derby.version>10.14.1.0</derby.version>
     <mariadb.version>2.5.0</mariadb.version>
     <mssql.version>6.2.1.jre8</mssql.version>
-- 
1.8.3.1

